src/Checkbox.js

```diff
@@ -7,15 +7,15 @@ import { Icon, Box } from '..'
 const Checkbox = props => {
   const { disabled, size } = props
   return (
-    <CheckBoxWrapper disabled={disabled}>
+    <CheckBoxWrapper disabled={disabled} theme={props.theme}>
```

Comment:
If this is a styled component, this shouldn't need the `theme` prop to be passed explicitly

Reply:
after replacing `extend` with `styled()`, this test was failing: https://github.com/pricelinelabs/design-system/blob/master/src/__tests__/Checkbox.js#L16

Reply:
I think that's probably a case where we can update the snapshot??